### DISCUSSION TITLE
External led strip

### DISCUSSION THREAD

**Discussion by thomytp** [jomjol/AI-on-the-edge-device#2884] [Q&A]
Hello,
I use for my watermater a external led strip. This works really good, but the strip did not turn off sometimes. 
the strip is then alway on.
What can I do? Thanks…

**Comment by caco3** [comment]
Please provide debug logs

**Comment by zorkakk** [comment]
Hi! Could someone please help me out and give some tips on how to use an external LED? I suppose I need to power it from an external 5V power supply? If yes, how do I wire it to the ESP to be able to control it and use it instead of the internal LED? Thank you!
@thomytp could you please share how you solved it?

**Reply by SybexX** [reply]
https://jomjol.github.io/AI-on-the-edge-device-docs/External-LED/
